packages/appium/test/cli-e2e-specs.js

```diff
@@ -15,6 +15,7 @@ const executable = path.join(cwd, 'packages', 'appium', 'build', 'lib', 'main.js
 
 describe('Driver CLI', function () {
   let appiumHome;
+  const localFakeDriverPath = path.resolve(cwd, 'packages', 'fake-driver');
```

Comment:
this resolution strategy will work only if running tests from the main appium directory. what if we're running them inside packages/appium? It might be better to do something like `path.resolve(__dirname, '..', '..', '..', 'fake-driver')` (with an extra '..' in the list because the transpiled code will run from `packages/appium/build`, not `packages/appium`)

Reply:
i'm ok if we leave this as it is right now, i understand you were just following the existing strategy.